src/processes/follow-topic-process.js

```diff
@@ -134,18 +134,30 @@ const followTopicProcess = async (job, done) => {
 
         await chat.create();
         const channelState = await chat.watch();
-
         if (channelState.messages.length === 0) {
           await chat.sendMessage({
             text: communityMessageFormat.message,
-            user_id: communityMessageFormat.user_id
+            user_id: communityMessageFormat.user_id,
+            is_auto_message: true
           });
 
           try {
             // await chat.stopWatching();
           } catch (error) {
             console.log('::: Error on stopWatching :::', error);
           }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's a commented out line of code here (`// await chat.stopWatching();`). If this line is not needed, it should be removed to avoid confusion. If it's needed in the future, it can be retrieved from version control history.

```diff
- // await chat.stopWatching();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->